.github/workflows/publish_build.yml

```
@@ -0,0 +1,84 @@
+          release_name: Release ${{ steps.version.outputs.version }}
+          body: |
+            Changes in this Release
+            ${{ steps.changelog.outputs.changelog }}
+          draft: false
+          prerelease: false
+          commitish: dev
+      - name: Build Distribution Packages
+        run: |
+          python setup.py bdist_wheel
+      - name: Prepare next Build version
+        run: echo "::set-output name=version::$(python setup.py --version)"
```

actionlint

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)